src/OrleansServiceBus/Providers/Streams/EventHub/EventHubConstants.cs

```diff
@@ -0,0 +1,12 @@
+namespace Orleans.ServiceBus.Providers
+{
+    internal class EventHubConstants
+    {
+        public readonly static string StartOfStream =
+#if NETSTANDARD
+            Microsoft.Azure.EventHubs.PartitionReceiver.StartOfStream;
+#else
+            Microsoft.ServiceBus.Messaging.EventHubConsumerGroup.StartOfStream;
+#endif
+    }
```

Comment:
Instead of creating a new class, I would create this as an internal field in `EventHubCheckpointer` class. Don't feel very strongly about it, so if there are no other changes required before the merge, feel free to ignore